src/repl/scala/tools/nsc/interpreter/IMain.scala

```diff
@@ -900,10 +896,7 @@ class IMain(initialSettings: Settings, protected val out: JPrintWriter) extends
                           |}
                           |""".stripMargin
 
-      import nme.{ INTERPRETER_IMPORT_WRAPPER => iw }
-
-      /** Adds a val that instantiates the wrapping class. */
-      def postwrap = s"}\nval $iw = new $iw\n"
+      def postwrap = s"}\n"
```

Comment:
Thanks for pointing that out @dwijnand . The Wrapper also doesn't need to be passed into the Imports generator. There is no more import wrapping and therefore no need for postwrap at all.